templates/krb5.conf.erb

```diff
@@ -15,7 +15,7 @@ admin_server = <%= @logging_admin_server %>
 krb524d = <%= @logging_krb524d %>
 <% end -%>
 <% end -%>
-<% if @default_realm or @dns_lookup_realm or @dns_lookup_kdc or @ticket_lifetime or @default_keytab_name or @forwardable or @allow_weak_crypto or @proxiable or @rdns -%>
+<% if @default_realm or @dns_lookup_realm or @dns_lookup_kdc or @ticket_lifetime or @default_ccache_name or @default_keytab_name or @forwardable or @allow_weak_crypto or @proxiable or @rdns or @default_tkt_enctypes or @default_tgs_enctypes -%>
```

Comment:
Adding this functionality is a guess, but I'm sure this would be the intentioned behaviour.
Without this using the added parameters wouldn't add them to the conf file if used alone.

Reply:
OK sounds sane.